internal/validations/clusterlogforwarder/inputs/validate_test.go

```diff
@@ -409,53 +409,4 @@ var _ = Describe("#Validate", func() {
 			Expect(clfStatus.Inputs["custom-app-group-limit"]).To(HaveCondition(loggingv1.ValidationCondition, true, loggingv1.ValidationFailureReason, "cannot have a negative limit"))
 		})
 	})
-	Context("when validating application input types", func() {
```

Comment:
Why are these test being removed? They are verifying admins improperly defining a namespace glob

Reply:
I found that these tests were duplicated and were already defined here: https://github.com/openshift/cluster-logging-operator/blob/4023485a589db9952b8b39ea557ab3d63aa67b37/internal/validations/clusterlogforwarder/inputs/validate_test.go#L127
Just a little clean up.